### PR TITLE
Add support to run nodevert using golang clusterloader

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds the config and modifies the script to support
running node-vertical using golang clusterloader.

### CODE REVIEW

File: openshift_scalability/nodeVertical.sh [hunk 1]
```diff
@@ -19,8 +41,15 @@ clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
...
+  golang_clusterloader
+elif [ "$TYPE" == "python" ]; then
+  python_clusterloader
+else
+  echo "$TYPE is not a valid option, available options: golang, python"
+  exit 1
+fi
 
 # sleeping again to gather steady-state metrics after environment is loaded
 long_sleep
```
Comment (mffiedler) [inline openshift_scalability/nodeVertical.sh]: clean() shouldn't be called for golang - the framework takes care of that.

File: openshift_scalability/nodeVertical.sh [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
-MY_CONFIG=config/nodeVertical.yaml
+TYPE=$2
+
+long_sleep() {
+  local sleep_time=180
+  echo "Sleeping for $sleep_time"
+  sleep $sleep_time
+}
+
+clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
```
Comment (mffiedler) [inline openshift_scalability/nodeVertical.sh]: duplicate definition of clean - also defined on line 39

File: openshift_scalability/nodeVertical.sh (hunk 1 above)
Comment (mffiedler) [inline openshift_scalability/nodeVertical.sh]: I believe there is no need to call clean for golang - the e2e framework should clean up the project.

**Comment by mffiedler** [comment]
When I run the golang version I get a test failure - am I doing something wrong.

**# export KUBECONFIG=/etc/origin/master/admin.kubeconfig
# /usr/libexec/atomic-openshift/extended.test --ginkgo.focus="Load cluster" --viper-config=./config/golang/nodeVertical**

/builddir/build/BUILD/atomic-openshift-git-0.7c71a2d/_output/local/go/src/github.com/openshift/origin/test/extended/cluster/cl.go:181
  should load the cluster [Suite:openshift] [BeforeEach]
  /builddir/build/BUILD/atomic-openshift-git-0.7c71a2d/_output/local/go/src/github.com/openshift/origin/test/extended/cluster/cl.go:180

  **Nov 29 20:41:26.654: Error parsing config: Unsupported Config Type ""**

**Comment by chaitanyaenr** [comment]
@mffiedler I think the config file needs to have .yaml extension. I have updated the PR, can you try again.

File: openshift_scalability/nodeVertical.sh [hunk 3]
```diff
@@ -16,17 +17,41 @@ long_sleep() {
 
 clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
 
-# sleeping to gather some steady-state metrics, pre-test
-long_sleep
+golang_clusterloader() {
+  # Export kube config
+  export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+  MY_CONFIG=config/golang/nodeVertical
```
Comment (mffiedler) [inline openshift_scalability/nodeVertical.sh]: Change to nodeVertical.yaml to match file

Comment (chaitanyaenr) [reply openshift_scalability/nodeVertical.sh]: golang clusterloader doesn't require a .yaml extension.

Comment (mffiedler) [reply openshift_scalability/nodeVertical.sh]: That's just strange that the viper config value and filename would not match.

File: openshift_scalability/nodeVertical.sh [hunk 4]
```diff
@@ -16,17 +17,41 @@ long_sleep() {
...
 
-# loading cluster based on yaml config file
-./cluster-loader.py --file=$MY_CONFIG
+python_clusterloader() {
+  MY_CONFIG=config/nodeVertical.yaml
+  ./cluster-loader.py --file=$MY_CONFIG
+}
 
-# sleeping again to gather steady-state metrics after environment is loaded
+long_sleep() {
```
Comment (mffiedler) [inline openshift_scalability/nodeVertical.sh]: Sorry, missed this before.  There are also duplicate long_sleep() in here.

Comment (chaitanyaenr) [reply openshift_scalability/nodeVertical.sh]: Sorry, didn't notice that I had the sleep function twice. Fixed.

File: openshift_scalability/nodeVertical.sh (hunk 3 above)
Comment (chaitanyaenr) [reply openshift_scalability/nodeVertical.sh]: I think it assumes that the viper config value has a .yaml extension.

**Comment by chaitanyaenr** [comment]
@mffiedler PTAL.

**Pull request merged**